app/blueprints/assessments/routes.py

```diff
@@ -514,6 +514,8 @@ def assessor_type(fund_short_name: str, round_short_name: str):
     if not (selected_assessments := request.form.getlist("selected_assessments")):
         abort(500, "Required selected_assessments field to be populated")
 
+    from_form = request.form.get("from_form")
```

Comment:
I wonder if something more explicit like `redirect_to_application_detail` would make the control flow read easier. I think originally `is_application` or `is_application_dashboard` which also sound like good options. 

Another option might be `action=application_overview` or an enum which would be more flexible but I suspect these are the only two that will be needed so thats likely overdoing it!

Reply:
Renamed to redirect_to_application_detail